README.md

```diff
@@ -6,82 +6,82 @@ OpenShift Application Platform
 [![Travis](https://travis-ci.org/openshift/origin.svg?branch=master)](https://travis-ci.org/openshift/origin)
 [![Jenkins](https://ci.openshift.redhat.com/jenkins/buildStatus/icon?job=devenv_ami)](https://ci.openshift.redhat.com/jenkins/job/devenv_ami/)
 [![Join the chat at freenode:openshift-dev](https://img.shields.io/badge/irc-freenode%3A%20%23openshift--dev-blue.svg)](http://webchat.freenode.net/?channels=%23openshift-dev)
+[![Licensed under Apache License version 2.0](https://img.shields.io/github/license/openshift/origin.svg?maxAge=2592000)](https://www.apache.org/licenses/LICENSE-2.0)
 
-OpenShift Origin is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment.  Origin adds developer and operations-centric tools on top of Kubernetes to enable rapid application development, easy deployment and scaling, and long-term lifecycle maintenance for small and large teams.
+***OpenShift Origin*** is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment.  Origin enables teams of all sizes to quickly develop, deploy, and scale applications, reducing the length of development cycles and operational effort.
```

Comment:
> OpenShift Origin is a distribution of Kubernetes" isn't sufficient to indicate that we extend Kube

I read those two sentences and it wasn't clear what origin is trying to be.  It could be nothing more than redistributing kube.

I personally still prefer the original sentence still.  The only thing that bothers me is it doesn't list build and test/ci/cd.  Although you can maybe assume those in develop.